### PR TITLE
Update the overall tmt description and examples

### PR DESCRIPTION
**Pull request by psss**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -9,28 +9,30 @@ Test Management Tool
 Description
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ``tmt`` python module and command line tool implement the L1
-and L2 Metadata Specification which allows to store all needed
-test execution data directly within a git repository. In this way
-it makes tests independent on any external test management system.
+The ``tmt`` python module and command line tool implement the
```
Comment (thrix) [inline README.rst:12]: `command-line` would be more correct

File: README.rst [hunk 2]
```diff
@@ -9,28 +9,30 @@ Test Management Tool
...
+Metadata Specification which allows to store all needed test
```
Comment (thrix) [inline README.rst:13]: `to store` -> `storing`

File: README.rst [hunk 3]
```diff
@@ -9,28 +9,30 @@ Test Management Tool
...
+execution data directly within a git repository. In this way it
```
Comment (thrix) [inline README.rst:14]: comma after `way`

File: README.rst [hunk 4]
```diff
@@ -9,28 +9,30 @@ Test Management Tool
...
-functionality has already been implemented.
+Plans are used to group tests and configure individual test steps
+defined by the L2 specification. They describe how to select tests
+for execution, how to provision the environment, how to prepare it
+for testing or how the test results should be reported.
+
+Stories, defined by the L3 specification, can be used to track
+implementation, test and documentation coverage for individual
+features or requirements. Thanks to this you can track everything
+at one place, including progress of your project implementation.
```
Comment (thrix) [inline README.rst:35]: `in one place`

Comment (thrix) [reply README.rst:35]: `the progress`

**Review by martinky82 — APPROVED** [review]
Ok from my point of view. Seems clarer and more coherrent.

**Comment by psss** [comment]
Thanks for the review. All feedback from @thrix should now be addressed.

**Review by thrix — APPROVED** [review]
LGTM

**Pull request merged**